tests/test_pages_auto.py

```diff
@@ -79,6 +79,13 @@
 )
 
 
+async def assert_expected_item(page, item):
+    assert await page.to_item() == item
+    # Ensure that all fields are sync.
+    for key in item.__slots__:
```

Comment:
Is there an "attrs" way to get the list of fields? Or maybe itemadapter way?

Reply:
:shrug:

Reply:
`ItemAdapter` uses `[a.name for a in attr.fields(item_class)]`, but I guess I can just use `ItemAdapter` itself, since it is a dependency already.